crates/virtio-queue-ser/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# v0.1.0
+
+This is the first release of the crate.
```

Comment:
I think we could also mention the version of `QueueState` that is compatible with `QueueStateSer` 0.1.0, similarly to what we did [here](https://github.com/rust-vmm/vm-superio/blob/main/crates/vm-superio-ser/CHANGELOG.md) for 0.1.0 and failed to do for 0.2.0.

Reply:
Mentioned the compatibility.